contrib/NeonVM/cell.c

```diff
@@ -403,3 +400,6 @@ void cell_freeCell(Cell *c)
     free(c);
 }
 
+#ifdef _MSC_VER
+#pragma warning(pop)
```

Comment:
This shouldn't be so far away from the push. What is it popping? No idea. You probably don't even need to push at all, or what might be better is to push the warning disable, #include the header file(s) you need, then immediately pop the warning. Of course, this applies to everywhere where you push to disable this warning, not just this file.

Reply:
This entire #pragma is really only intended as a work-around until I can figure out the correct way to handle this in the Windows build.  Otherwise, I see your point and will change these in future commits.